sentry_sdk/integrations/redis/_sync_common.py

```diff
@@ -64,18 +69,37 @@ def sentry_patched_execute_command(self, name, *args, **kwargs):
         if integration is None:
             return old_execute_command(self, name, *args, **kwargs)
 
-        description = _get_span_description(name, *args)
+        cache_properties = _compile_cache_span_properties(
+            integration, name, args, kwargs
+        )
+
+        cache_span = None
+        if cache_properties["is_cache_key"] and cache_properties["op"] is not None:
+            cache_span = sentry_sdk.start_span(
+                op=cache_properties["op"],
+                description=cache_properties["description"],
+            )
+            cache_span.__enter__()
```

Comment:
Would it make more sense to first start the db span and having the cache span be nested in the db span? Conceptually it feels like the cache lookup is the first step of the db span, rather than the other way around

Reply:
We agreed on having first the cache and then the db span. Because it could (like in Django) that you access the cache and inside the cache span there is a db call, or redis call, or filesystem read, or wherever your cache is)

Reply:
So we will keep it like this for now. If users complain that it is the other way round, then lets reevaluate.